opsmngr/opsmngr.go

```diff
@@ -123,6 +124,7 @@ func NewClient(httpClient *http.Client) *Client {
 	c.GlobalAPIKeys = &GlobalAPIKeysServiceOp{Client: c}
 	c.GlobalAPIKeysWhitelist = &GlobalAPIKeyWhitelistsServiceOp{Client: c}
 	c.MaintenanceWindows = &MaintenanceWindowsServiceOp{Client: c}
+	c.PerformanceAdvisor = &PerformanceAdvisorServiceOp{Client: c}
```

Comment:
is there any difference between the atlas and Ops Manager implementation? any reason we can't use `&atlas.PerformanceAdvisorServiceOp{Client: c}` here? I just can't see any difference right now

Reply:
The path is different

ops manager `GET /groups/{PROJECT-ID}/hosts/{HOST-ID}/performanceAdvisor/suggestedIndexes`
atlas `GET /groups/{GROUP-ID}/processes/{PROCESS-ID}/performanceAdvisor/suggestedIndexes`